include/zephyr/bluetooth/conn.h

```diff
@@ -216,7 +216,17 @@ struct bt_conn *bt_conn_ref(struct bt_conn *conn);
  */
 void bt_conn_unref(struct bt_conn *conn);
 
-/** @brief Iterate through all existing connections.
+/** @brief Iterate through all bt_conn objects.
+ *
+ * Iterates trough all #bt_conn objects that are alive in the Host allocator.
```

Comment:
A doxygen question:  What do the `#` in `#bt_conn` do? Is it shorthand for `@ref bt_conn`?

Reply:
Yes. Shall I change it to `@ref`?

Reply:
Personally I kind of like the `#`, but on other hand `@ref` is what we use everywhere else, and it would be nice to be consistent, right? If someone went ahead and used `\ref` I'd probably also suggest to use `@ref` instead.